gomint-server/src/main/java/io/gomint/server/network/packet/PacketTransfer.java

```diff
@@ -0,0 +1,26 @@
+package io.gomint.server.network.packet;
+
+import io.gomint.jraknet.PacketBuffer;
+import io.gomint.server.network.Protocol;
+
+public class PacketTransfer extends Packet {
+
+    public String address;
```

Comment:
No public fields. Use Lombok @Data annotation on class and use setXYZ() and getXYZ() methods

Reply:
Okay ^^